TorchSharp/TypeGeneration.cs

```diff
@@ -142,96 +142,76 @@ public void Fill (byte value)
     /// </remarks>
     public class ByteTensor : IDisposable {
```

Comment:
Do we need this file in the repo, given that it is auto-generated?

Reply:
This file is in master that's why it is also here but I agree.